### ISSUE TITLE
torch dynamic shape

### ISSUE THREAD

**Issue by heouchen** [aiff22/MAI-2025-Workshop#1]
Could you please help me provide torch code to generate dynamic size tflite? I tried the following solution but failed.

try torch model to tflite with dynamic input shape：

    model = UNetSeeInDark_prune_raw2rgb2(prune=5, in_channels=4, out_channels=3)
    model = load_compiled_model(model, '/home/chen/work/cvpr_isp/result/logs_20250319_014411/unet32_0271_psnr26.664.pth', 'cpu')
    model.eval()

    # 转换为支持动态尺寸的TFLite模型
    # 确保输入张量的形状符合动态形状约束
    sample_inputs = (torch.randn(1, 4, 4096, 4096),)  # 示例输入，确保第一个维度是4096的倍数

    # 定义动态尺寸
    _height = Dim('_height', min=1, max=256)
    _width = Dim('_width', min=1, max=256)
    height = 32*_height
    width = 32*_width
    # import pdb; pdb.set_trace()
    # 设置动态形状配置
    dynamic_shapes = {
        'x': {  # 使用输入参数名称
            2: height,  
            3: width
        }
    }

    # 使用动态形状转换模型
    edge_model = ai_edge_torch.convert(
        model.eval(),
        sample_inputs,
        dynamic_shapes=dynamic_shapes
    )
    edge_model.export("memory_attention_dynamic.tflite")
